### PR TITLE
Fix errors in solutions in session 2

### PR DESCRIPTION
**Pull request by AlinderS**
I found two errors in the suggested solutions for the exercises in session 2. This PR fixes them both.

### CODE REVIEW

File: 2-numpy/exercises.ipynb [hunk 1]
```diff
@@ -920,7 +920,8 @@
     "    \n",
     "gt.show_in_notebook(display_length=10)\n",
     "    \n",
-    "gt.add_column((gt[\"vy\"] * gt[\"x\"] - gt[\"vx\"] * gt[\"y\"]), name=\"Lz\")    \n",
+    "gt.add_column((gt[\"vy\"] * gt[\"x\"] - gt[\"vx\"] * gt[\"y\"]), name=\"Lz\")\n",
+    "gt.add_column((np.sqrt(gt[\"x\"]**2 + gt[\"y\"]**2)), name=\"r\")\n",
```
Comment (eerovaher) [inline 2-numpy/exercises.ipynb:924]:
```suggestion
    "gt.add_column(np.hypot(gt[\"x\"], gt[\"y\"]), name=\"r\")\n",
```

File: 2-numpy/exercises.ipynb [hunk 2]
```diff
@@ -920,7 +920,8 @@
     "    \n",
     "gt.show_in_notebook(display_length=10)\n",
     "    \n",
-    "gt.add_column((gt[\"vy\"] * gt[\"x\"] - gt[\"vx\"] * gt[\"y\"]), name=\"Lz\")    \n",
+    "gt.add_column((gt[\"vy\"] * gt[\"x\"] - gt[\"vx\"] * gt[\"y\"]), name=\"Lz\")\n",
+    "gt.add_column(np.hypot(gt[\"x\"], gt[\"y\"]), name=\"r\")\n",
```
Comment (eerovaher) [inline 2-numpy/exercises.ipynb:924]: The code here isn't really using any noteworthy features of `QTable.add_column()`, so we might as well edit it some more:
```suggestion
    "gt[\"Lz\"] = gt[\"vy\"] * gt[\"x\"] - gt[\"vx\"] * gt[\"y\"])\n",
    "gt[\"r\"] = np.hypot(gt[\"x\"], gt[\"y\"])\n",
```

Comment (AlinderS) [reply 2-numpy/exercises.ipynb:924]: I think it is worth using `QTable.add_column()` for two reasons;
1. It makes it very clear what the code is doing, and that a `Table` is not just a dictionary.
2. It informs students that the method exists by presenting an example.

Comment (eerovaher) [reply 2-numpy/exercises.ipynb:924]: A `Table` _is_ a dictionary of columns with some additional convenience methods and the ability to store extra metadata, even if it isn't a subclass of `dict`. I wouldn't bother using `Table.add_columns()` unless I really cared about column ordering.

Comment (AlinderS) [reply 2-numpy/exercises.ipynb:924]: Fair point, I accept.

**Pull request merged**